apps/giftify/src/app/shared/gift.model.ts

```diff
@@ -0,0 +1,9 @@
+export interface Gift {
+  name: string;
+  age: {
+    ageMin: number;
+    ageMax: number;
```

Comment:
Trochę dziwnie będzie wyglądać `gift.age.ageMin`, bo "age" jest niepotrzebnie powtórzone. Nie lepiej byłoby to nazwać po prostu `min` i `max`?
```suggestion
    min: number;
    max: number;
```